microprofile/6.1/config/3.1/23.0.0.12-MicroProfile-Config-3.1-Java11-TCKResults.adoc

```diff
@@ -0,0 +1,487 @@
+:page-layout: certification 
+= TCK Results
+
+As required by the https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License], following is a summary of the TCK results for releases of MicroProfile Config 3.1
+
+== Open Liberty 23.0.0.12 - MicroProfile Config 3.1 Certification Summary (Java 11)
+
+* Product Name, Version and download URL (if applicable):
++
+https://public.dhe.ibm.com/ibmdl/export/pub/software/openliberty/runtime/release/23.0.0.12/openliberty-23.0.0.12.zip[Open Liberty 23.0.0.12]
+
+* Specification Name, Version and download URL:
++
+https://github.com/eclipse/microprofile-config/tree/3.1[MicroProfile Config 3.1 Specification]
+
+* (Optional) TCK Version, digital fingerprint and download URL:
++
+https://repo1.maven.org/maven2/org/eclipse/microprofile/config/microprofile-config-tck/3.1/microprofile-config-tck-3.1.jar[MicroProfile Config 3.1 TCK]
++
+SHA-1: `d0579ab0f0f4f06e25161b60838834b3003b51d6`
++
+SHA-256: `1b7bde78c8fbd6a297cd6ed33aa98aca21cab82ec198fb8b0a3ac1747136b340`
+
+* Public URL of TCK Results Summary:
++
+xref:23.0.0.12-MicroProfile-Config-3.1-Java11-TCKResults.adoc[TCK results summary]
+
+
+* Java runtime used to run the implementation:
++
+Java 11: OpenJDK Runtime Environment (11.0.21+9)
```

Comment:
So we are doing a mix of Open JDK and Semeru in our results?

Reply:
Yes, from a TCK perspective the exact vendor doesn't matter

Reply:
Of course it doesn't matter, but we use Semeru for Core Profile, so it would make sense to use Semeru for MicroProfile as well.  As you stated the results appear to be missing from the build for some reason, so I guess that is what we have.